tests/framework/test_output.xml

```diff
@@ -684,7 +684,7 @@
       <Output class="OutStreams" type="Print">test</Output>
       <Output class="OutStreams" type="Print">fullprint_Pointset</Output>
       <Output class="OutStreams" type="Plot">2DHistoryPlot2</Output>
-      <Output class="OutStreams" type="Plot">test</Output>
+      <Output class="OutStreams" type="Plot">testPlot</Output>
```

Comment:
why are you modifying this name? if there is no specific reason can you revert it? (just to avoid unneeded file histories)

Reply:
we prevent Entities from having duplicate naming; there's an OutStream Print called `test` already in this file. Since this is the only file that had that problem in all 700 regression tests, I figured it wasn't a big deal to comply with that.